Contest_API.md

```diff
@@ -749,7 +749,7 @@ Properties of contest objects:
 | name                         | string          | Short display name of the contest.
 | formal\_name                 | string ?        | Full name of the contest. Defaults to value of `name`.
 | start\_time                  | TIME ?          | The scheduled start time of the contest, may be `null` if the start time is unknown or the countdown is paused.
-| countdown\_pause\_time       | RELTIME ?       | The amount of seconds left when countdown to contest start is paused. At no time may both `start_time` and `countdown_pause_time` be non-`null`.
+| countdown\_pause\_time       | RELTIME ?       | The number of seconds left when the countdown to the contest start was paused. At no time may both `start_time` and `countdown_pause_time` be non-`null`.
```

Comment:
But this is not a number, it's a `RELTIME`. The old wording; "amount of seconds" is clearly wrong, but maybe "amount of time" is a better fix than "number of seconds"?

There should be a "if the contest is paused" in there. Maybe something like this?:

```suggestion
| countdown\_pause\_time       | RELTIME ?       | The amount of time left when the countdown to the contest start was paused, if the contest is paused, otherwise `null`.
```